src/api/providers/__tests__/fireworks.test.ts

```diff
@@ -0,0 +1,786 @@
+// npx jest src/api/providers/__tests__/fireworks.test.ts
```

Comment:
I haven't looked too deeply yet, but why is this file so much longer than most of the other tests in `/src/api/providers/__tests__`? Just wondering if there are some unnecessary tests.